include/random_walks/gaussian_accelerated_billiard_walk.hpp

```diff
@@ -67,9 +68,15 @@ struct GaussianAcceleratedBilliardWalk
     struct Walk
     {
         typedef typename Polytope::PointType Point;
-        typedef typename Polytope::DenseMT DenseMT;
+        typedef typename Polytope::MT MT;
+        typedef typename Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> DenseMT;
         typedef typename Polytope::VT VT;
         typedef typename Point::FT NT;
+        using AA_type = std::conditional_t< std::is_same_v<MT, typename Eigen::SparseMatrix<NT, Eigen::RowMajor>>, typename Eigen::SparseMatrix<NT>, DenseMT >; 
+        // AA is sparse colMajor if MT is sparse rowMajor, and Dense otherwise
```

Comment:
Is it better to move the comment before the definition? (The same for the comment below).